### PR TITLE
[skip ci] Improve ShowCollections in milvus.proto

### PR DESCRIPTION
**Pull request by nameczz**
Signed-off-by: nameczz <zizhao.chen@zilliz.com>
Issues: #7449

### CODE REVIEW

File: internal/proto/milvus.proto [hunk 1]
```diff
@@ -187,34 +187,62 @@ message GetCollectionStatisticsResponse {
   repeated common.KeyValuePair stats = 2;
 }
 
-
+/*
+* This is for ShowCollectionsRequest type field.
+*/
 enum ShowType {
+  // Will return all colloections without inMemory_percentages
```
Comment (DragonDriver) [inline internal/proto/milvus.proto:194]: typo "colloections"
Just tell users "Will return all collections in Milvus".

File: internal/proto/milvus.proto [hunk 2]
```diff
@@ -187,34 +187,62 @@ message GetCollectionStatisticsResponse {
...
+  // Not useful for now
+  common.MsgBase base = 1;
+  // Not useful for now
   string db_name = 2;
+  // Not useful for now
   uint64 time_stamp = 3;
+  // Decide return Loaded collections or All collections(Optional)
   ShowType type = 4;
-  repeated string collection_names = 5; // show collection in querynode, showType = InMemory
+  // When type is InMemory, will return these collection names inMemory_percentages.(Optional)
```
Comment (DragonDriver) [inline internal/proto/milvus.proto:212]: these collections' inMemory_percentages

File: internal/proto/milvus.proto [hunk 3]
```diff
@@ -187,34 +187,62 @@ message GetCollectionStatisticsResponse {
...
 }
 
+/*
+* Create partition in created collection.
+*/
 message CreatePartitionRequest {
-  common.MsgBase base = 1; // must
+  // Contain error_code and reason
+  common.MsgBase base = 1;
+  // Contain error_code and reason
```
Comment (DragonDriver) [inline internal/proto/milvus.proto:240]: The db name in Milvus

**Review by longjiquan — APPROVED** [review]
thx for improving this

**Pull request merged**